app/services/PackageTypes.scala

```diff
@@ -33,5 +33,5 @@ object PackageTypes {
 
   val packageTypeCodeMap: Map[String, PackageType] = all.map(packageType => (packageType.code, packageType)).toMap
 
-  def findByCode(code: String): PackageType = packageTypeCodeMap(code)
+  def findByCode(code: String): Option[PackageType] = packageTypeCodeMap.get(code)
```

Comment:
Why does this return Option now?   Do we need to support unknown codes?

Reply:
This stayed after some previous version. Thank you for spotting this.